packages/react-pose/src/_tests/index.test.tsx

```diff
@@ -390,25 +384,21 @@ test('PoseGroup: Provides group props to children on mount', () => {
   let x = 0;
   let y = 0;
 
-  const ChildWithGroupProps = posed(({ innerRef, hostRef, text, ...props }) => (
-    <div ref={innerRef} {...props}>{`text: ${text}`}</div>
-  ))({});
+  const ChildWithGroupProps = posed(
```

Comment:
I think this test could not use `forwardRef` at all - for its simplicity (it doesnt matter to it), I've written it this way because of other issue that in the meantime I've fixed - so now it should be possible to simplify it, what should matter for it is just a property resolver in `enter` pose (which has to be added to the config now)